mythplugins/mythmusic/mythmusic/visualize.cpp

```diff
@@ -1315,19 +1314,23 @@ void Spectrum::resize(const QSize &newsize)
 
     m_size = newsize;
 
-    m_analyzerBarWidth = m_size.width() / 64;
+    m_analyzerBarWidth = m_size.width() / 128;
 
     if (m_analyzerBarWidth < 6)
         m_analyzerBarWidth = 6;
```

Comment:
Possibly replace this with something like:
```
m_analyzerBarWidth =
    std::max(m_size.width() / kNumBarsWanted, kMinBarSize);
```
and define the two constants in the Spectrum class with:
``` 
static constexpr int kNumBarsWanted {128};
static constexpr int kMinBarSize {6};
```
I know there are close to 15,000 other magic numbers in the code, so I'll leave it to you whether to change this. I do think this type of change makes the code more readable.

Reply:
Personally, for a number used in 1 place, the number is easier to read than words.  Because if it is words I have to go lookup where the words resolve to a number.  So reading 128 and 6 right here is easier for me.

But certainly numbers used in more than 1 place should be words instead.

Reply:
You understand the code, so its not surprising that you find the numbers easier. For someone reading the code for the first time, the descriptive constant names would be easier.